### PR TITLE
[quant] qtensor resize

### PR DESCRIPTION
**Pull request by z-a-f**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #36450 [quant] quantized reflection_pad1d
* #36449 [quant] Generalizing _calculate_dynamic_qparams in quantized test
* **#36442 [quant] qtensor resize**

Differential Revision: [D20984080](https://our.internmc.facebook.com/intern/diff/D20984080)

### CODE REVIEW

**Comment by jerryzh168** [comment]
Can you check if we can add this to graph mode? is this also compiled to `CallFunction`?

**Comment by z-a-f** [comment]
> Can you check if we can add this to graph mode? is this also compiled to `CallFunction`?

It doesn't have the CallFunction. Also, added an entry to quantization.cpp

File: torch/csrc/jit/passes/quantization.cpp [hunk 1]
```diff
@@ -118,6 +118,7 @@ std::vector<std::string> _single_input_general_aten_funcs = {
     "upsample_bicubic2d",
     "dropout",
     "reshape",
+    "resize_",
```
Comment (jerryzh168) [inline torch/csrc/jit/passes/quantization.cpp:121]: what about "resize"?

Comment (z-a-f) [reply torch/csrc/jit/passes/quantization.cpp:121]: `resize` is a python module, and script does not see it:

```python
class Model(torch.nn.Module):
  def __init__(self, sz):
    super(Model, self).__init__()
    self.sz = sz
  def forward(self, x):
    return x.resize(self.sz)
torch.jit.script(Model(42))
```

will throw an error:

```console
RuntimeError: 
Tried to access nonexistent attribute or method 'resize' of type 'Tensor'.:
  File "<ipython-input-11-85bfdcc7b088>", line 6
  def forward(self, x):
    return x.resize(self.sz)
           ~~~~~~~~ <--- HERE
```

Comment (jerryzh168) [reply torch/csrc/jit/passes/quantization.cpp:121]: I see, looks like non-inplace resize is deprecated: https://github.com/pytorch/pytorch/blob/master/torch/tensor.py#L351

could you add a comment here for it?

**Review by jerryzh168 — APPROVED** [review]

**Pull request closed without merging**